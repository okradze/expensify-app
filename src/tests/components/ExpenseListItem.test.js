import React from 'react';
import ExpenseListItem from '../../components/ExpenseListItem';
import { shallow } from 'enzyme';

test('should render expense item correctly', () => {
    const wrapper = shallow(<ExpenseListItem id='1' description='Rent' note='' amount={125000} createdAt={0} />);
    expect(wrapper).toMatchSnapshot();
});