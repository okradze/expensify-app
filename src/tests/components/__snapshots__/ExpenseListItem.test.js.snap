// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render expense item correctly 1`] = `
<div>
  <Link
    replace={false}
    to="/edit/1"
  >
    <h3>
      Rent
    </h3>
  </Link>
  <p>
    $1,250.00
    â€”
    January 1st, 1970
  </p>
</div>
`;
